src/main/java/it/gov/pagopa/pu/fileshare/service/ingestion/IngestionFlowFileServiceImpl.java

```diff
@@ -15,18 +15,23 @@ public class IngestionFlowFileServiceImpl implements IngestionFlowFileService {
   private final UserAuthorizationService userAuthorizationService;
   private final FileService fileService;
   private final String validIngestionFlowFileExt;
+  private final String ingestionFlowFilePath;
 
   public IngestionFlowFileServiceImpl(
     UserAuthorizationService userAuthorizationService, FileService fileService,
-    @Value("${uploads.ingestion-flow-file.valid-extension}") String validIngestionFlowFileExt) {
+    @Value("${uploads.ingestion-flow-file.valid-extension}") String validIngestionFlowFileExt,
+    @Value("${folders.ingestion-flow-file.path}") String ingestionFlowFilePath
+    ) {
     this.userAuthorizationService = userAuthorizationService;
     this.fileService = fileService;
     this.validIngestionFlowFileExt = validIngestionFlowFileExt;
+    this.ingestionFlowFilePath = ingestionFlowFilePath;
```

Comment:
the paths should change based on the IngestionFlowFileType.
let's change the application.yml to have the following structure:
```
folders:
   ingestion-flow-file-type:
      <Enum name>: <relative path>
```

You could read this inside a @ConfigurationProperties class using prefix `folders` (FoldersConfig) having at least a property `ingestionFlowFileType` of type Map<IngestionFlowFileType, String>
(you could also add the other folders properties in order to use the same classe to read their values